source/Xunit.Gherkin.Quick/Table.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Gherkin.Ast;
+
+namespace Xunit.Gherkin.Quick
+{
+    public class Table : DataTable
```

Comment:
I don't see a need for this table. Please remove and just use DataTable directly.

Reply:
I'm not planning to make this framework backwards compatible with specflow. If I decide to do so, that will become completely different concern and I will have to maintain that compatibility.
So, I don't see a need for table class. Let's stick to DataTable.
Besides, `Quick` word was used to mean lightweight. I don't see a necessity to introduce new classes without need.

Reply:
We may need a wrapper for other, more clear design reasons in the future, but then we'll do it the way we really need. For now, YAGNI.